services/manifest-repo-export-service/pkg/cmd/server.go

```diff
@@ -112,6 +117,21 @@ func Run(ctx context.Context) error {
 		return err
 	}
 
+	var releaseVersionLimit uint64
+	if val, exists := os.LookupEnv("KUBERPULT_RELEASE_VERSIONS_LIMIT"); !exists {
```

Comment:
please change this so that the env var is required.
Otherwise we have 2 defaults: 1 in the helm chart and one in the code here. We should have only one.